assets/js/selfoss-ui.js

```diff
@@ -102,11 +102,13 @@ selfoss.ui = {
                     <hr />
                 </div>
 
-                <div class="nav-toolbar">
-                    <button id="nav-refresh" title={selfoss.ui._('refreshbutton')} aria-label={selfoss.ui._('refreshbutton')} accesskey="r"></button>
-                    <button id="nav-settings" title={selfoss.ui._('settingsbutton')} aria-label={selfoss.ui._('settingsbutton')} accesskey="t"></button>
-                    <button id="nav-logout" title={selfoss.ui._('logoutbutton')} aria-label={selfoss.ui._('logoutbutton')} accesskey="l"></button>
-                    <button id="nav-login" title={selfoss.ui._('loginbutton')} aria-label={selfoss.ui._('loginbutton')} accesskey="l"></button>
+                <div class="nav-toolbar" id="nav_toolbar" aria-label={selfoss.ui._('nav_toolbar')} role="toolbar">
+                    <ul aria-labelledby="nav_toolbar">
```

Comment:
Will not this be read twice? Would not it be sufficient to add `aria-label` and `role` here?

Reply:
Actually, does toolbar even make sense here? The buttons are grouped together mostly because there was not anywhere else to put them. Reading the [practices document](https://www.w3.org/TR/wai-aria-practices/#toolbar), it seems to me that it is mostly meant as an encapsulation for a large number of buttons, which is not really the case  here. Quite the contrary, users will want to be quickly able to access the settings page, not [rove](https://www.w3.org/TR/wai-aria-practices/#kbd_roving_tabindex) through a toolbar. The place where it makes sense to me is the entry action bar, where having to tab through all the sharing buttons while trying to get to the next entry would be annoying. We should implement roving behaviour there.